app/assets/javascripts/discourse/app/components/basic-topic-list.hbs

```diff
@@ -1,6 +1,6 @@
 <ConditionalLoadingSpinner @condition={{this.loading}}>
   {{#if this.topics}}
-    {{#if this.currentUser.use_glimmer_topic_list}}
+    {{#if this.currentUser.canUseGlimmerTopicList}}
```

Comment:
What about anon users? I think we need the auto feature flag to work for them too 🤔

Reply:
for now you need **both** no outdated customizations and to belong to the experimental group for the gjs version to be enabled. so it's fine now, but yes - I'll have to move this check when changing the logic for a wider gjs topic-list rollout.